src/Molecules/Editable/Editable.jsx

```diff
@@ -46,26 +46,35 @@ const Editable = ({
   label = '',
   Type = Input,
   value = '',
+  labelSize = 'title',
+  descriptionSize = 'title',
+  descriptionFontFamily = '',
   ...props
 }) => (
   edit ? <FormControl label={label} Type={Type} value={value} {...props}/> :
   <FlexBox onClick={() => onEdit(true)} gap={1} flexDirection="column">
     {!!label &&
-        <Typo fontSize="title" fontWeight="title" fontFamily="title">
+        <Typo fontSize={labelSize} fontWeight="title" fontFamily="title">
           {label}
         </Typo>
     }
-    <Description value={value} Type={Type} {...props}/>
+    <Description value={value} Type={Type} descriptionSize={descriptionSize} descriptionFontFamily={descriptionFontFamily} {...props}/>
   </FlexBox>
 )
 
-const DefaultDescription = ({ Type, value, children }) =>
+const DefaultDescription = ({
+  Type,
+  value,
+  descriptionSize,
+  descriptionFontFamily,

```

Comment:
We can call it `size` and `fontFamily` since we're already in the `Description` component.
And I think it's better to set the default values for these here and not in `Editable`